reactor-core/build.gradle

```diff
@@ -212,7 +212,7 @@ task testNoMicrometer(type: Test, group: 'verification') {
 //inherit basic test task + common configuration in root
 //always depend on testStaticInit, skip testNG on Travis, skip loops when not releasing
 //note that this way the tasks can be run individually
-test {
+check {
```

Comment:
isolate in separate commit (in case of merge-rebase) or separate pre-requisite PR

Reply:
submitted as #1776